Automattic/jetpack
# Remove Jetpack Blocks section from At a Glance page

This PR removes the section about Jetpack Blocks on the At a Glance Page to reduce clutter

#### Changes proposed in this Pull Request:

Remove Jetpack Blocks section from At A Glance and delete the file associated with that component

#### Jetpack product discussion

Context: 1198218726984184-as-1203627564453661/f

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

1. Check out this branch: `git switch remove/jetpack-blocks-section-at-a-glance`
2. Start up your local environment
3. Go to `/wp-admin/admin.php?page=jetpack#/dashboard`
4. Confirm that the Jetpack Blocks section is gone
![image](https://user-images.githubusercontent.com/65001528/210404231-57fbf480-906c-4425-9e27-34f03c7824f5.png)
5. Make sure there are no errors in the console relating to these changes
